docs/content/manual/_incoming/2263_Unable_to_stop_VM_which_in_starting_state.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Unable to stop VM which in starting state
+category: UI
+tags: VM, p2, acceptance
+---
+Ref: https://github.com/harvester/harvester/issues/2263
+
+
+### Verify Steps:
+1. Install Harvester with any nodes
+1. Create an Windows iso image for VM creation
+1. Create the Windows VM by using the iso image
```

Comment:
Maybe we could add this picture?:
https://github.com/harvester/harvester/issues/1730#issuecomment-1063878424

Maybe mentioning that we must select the `os-type` to be `Windows`?

Reply:
No it is different issue, we use Windows VM in here just for the long starting state.